themes/startwords/assets/scss/article/_figure.scss

```diff
@@ -0,0 +1,29 @@
+/* figures (images, diagrams) in articles */
+
+body.article figure {
+    margin: rem(30px) 0;
+
+    img {
+        display: block;
+        margin: 0 auto;
+        
+        &.landscape {
+            width: 100%;
```

Comment:
the designs don't specify; it looks to me like they should always be sized to fit the text container. since people should always be providing high enough quality versions to generate derivatives for all screens, I don't think we would have images small enough to naturally be less wide than the viewport, if that makes sense.